tests/starknet/starkli.rs

```diff
@@ -0,0 +1,195 @@
+use std::io::Write;
+
+use anyhow::{anyhow, Error};
+use clap::Parser;
+use starkli::{
+    account::{
+        AccountConfig, AccountVariant, DeploymentStatus, OzAccountConfig,
+        UndeployedStatus,
+    },
+    signer::AnySigner,
+    utils::{Cli, Subcommands},
+};
+use starknet::{
+    core::types::{contract::SierraClass, PriceUnit},
+    signers::{LocalWallet, Signer, SigningKey},
+};
+use starknet_crypto::Felt;
+
+#[allow(dead_code)]
+pub struct Starkli {
+    pub rpc: String,
+    account_folder: String,
+    prefunded_account: String,
+    persist_logger: bool,
+}
+
+#[allow(dead_code)]
+pub enum PreFundedAccount {
+    Katana,
+    Sepolia,
+}
+
+const ACCOUNT: &str = "account.json";
+const COMPILED_ACCOUNT: &str = "target/dev/account_Account.contract_class.json";
+const KEY: &str = "key.json";
+const PASSWORD: &str = "password";
+
+#[allow(dead_code)]
+impl Starkli {
+    pub fn new(
+        rpc: &str,
+        account_folder: &str,
+        prefunded_account: PreFundedAccount,
+    ) -> Self {
+        let prefunded_account = match prefunded_account {
+            PreFundedAccount::Katana => "katana-0".to_string(),
+            PreFundedAccount::Sepolia => "TODO".to_string(),
```

Comment:
If this TODO is not being addressed in this PR: let's maybe fail fast here, e.g. with `unimplemented!()`.